.cicd/build.sh

```diff
@@ -34,7 +34,10 @@ else # Linux
         CMAKE_EXTRAS="$CMAKE_EXTRAS -DLLVM_DIR=\"/opt/rh/llvm-toolset-7.0/root/usr/lib64/cmake/llvm\""
     elif [[ "$IMAGE_TAG" == 'ubuntu-18.04-unpinned' ]]; then
         CMAKE_EXTRAS="$CMAKE_EXTRAS -DCMAKE_CXX_COMPILER=\"clang++-7\" -DCMAKE_C_COMPILER=\"clang-7\" -DLLVM_DIR=\"/usr/lib/llvm-7/lib/cmake/llvm\""
+    elif [[ "$IMAGE_TAG" == 'centos-8-pinned' ]]; then
+        CMAKE_EXTRAS="$CMAKE_EXTRAS -DPostgreSQL_TYPE_INCLUDE_DIR=/usr/include/pgsql"
```

Comment:
I believe this is needed because the `postgresql-server-devel` package isn't being installed. To reduce clutter here you could consider installing that package that instead, but it's not a blocker by any means

Reply:
removed this part and added `postgresql-server-devel` package